PWGJE/Tasks/ChJetTriggerQATask.cxx

```diff
@@ -198,11 +200,15 @@ struct ChJetTriggerQATask {
           spectra.fill(HIST("globalP_tpcglobalPDiff_withoutcuts_phirestrict"), track.p(), track.p() - originalTrack.tpcInnerParam());
         }
 
+        spectra.fill(HIST("ptphiAllTracks"), track.pt(), track.phi());
+
         if (!jetderiveddatautilities::selectTrack(track, trackSelection)) {
           continue;
         }
 
-        bool bDcaCondition = (abs(track.dcaZ()) < dcaZ_cut) && (abs(track.dcaXY()) < dcaXY_multFact * DcaXYPtCut(track.pt()));
+        spectra.fill(HIST("ptphiQualityTracks"), track.pt(), track.phi());
+
+        bool bDcaCondition = (fabs(track.dcaZ()) < dcaZ_cut) && (fabs(track.dcaXY()) < dcaXY_multFact * DcaXYPtCut(track.pt()));
```

Comment:
Consider using `std::abs` everywhere to save yourself these surprises.

Reply:
@vkucera Thank you, it is another good option.
``fabs`` should work in the same way as ``std::abs``

Do you have rights to approve the code?

Reply:
No. If you click on "pending reviewers" below, you will se the list of code owners who can approve your PR.